src/Event/EnhancedEcommerceEvents.php

```diff
@@ -8,9 +8,9 @@
 final class EnhancedEcommerceEvents {
 
   /**
-   * Event fired after mapping a commerce product to a product
-   * represented in the domain of Enhanced Ecommerce. This allows
-   * to change the mapping of fields or enhance products with custom
+   * Event fired after mapping a commerce product to a product.
+   *
+   * This allows to change the mapping of fields or enhance products with custom
```

Comment:
Do you agree with this change ? 

The first comment must be a one line comment, the second can be multiline (drupal 8 coding standard).

Reply:
Yeah it's quite a challenge.

What about this ?

```
  /**
   * Allows to alter the GTM Product field mapping.
   *
   * Allows alteration of field mapping from Commerce product to a GTM Product.
   * Use this event to alter data Product before it gets pushed to data layer.
```

@wanze